package.json

```diff
@@ -258,11 +258,8 @@
     "ini": "^1.3.8",
     "is-installed-globally": "^0.4.0",
     "wait-on/axios": "^0.27.2",
-    "**/request/qs": "^6.11.0",
     "grunt/minimatch": "^3.1.2",
-    "**/globule/minimatch": "^3.1.2",
```

Comment:
Can you explain why you removed these resolutions? These were added to patch vulnerable nested dependencies